src/frontend/komponenter/Behandling/Inngangsvilkår/vilkår.ts

```diff
@@ -0,0 +1,63 @@
+import { IStatsborgerskap } from '../../../typer/personopplysninger';
+
+export interface IInngangsvilkår {
+    medlemskap: IMedlemskap;
+    vurderinger: IVurdering[];
+}
+
+export interface IVurdering {
+    id: string;
+    resultat: VilkårResultat;
+    behandlingId: string;
+    vilkårType: VilkårType;
+    begrunnelse: string;
+    unntak: string;
+    endretAv: string;
+    endretTid: string;
+}
+
+export interface IMedlemskap {
+    søknadGrunnlag: IMedlemskapSøknadGrunnlag;
+    registerGrunnlag: IMedlemskapRegisterGrunnlag;
+}
+
+export interface IMedlemskapSøknadGrunnlag {
+    bosattNorgeSisteÅrene: boolean;
+    oppholderDuDegINorge: boolean;
+    utenlandsopphold: IUtenlandsopphold[];
+}
+
+export interface IMedlemskapRegisterGrunnlag {
+    nåværendeStatsborgerskap: string[];
+    oppholdstatus: IOppholdstatus[];
+    statsborgerskap: IStatsborgerskap[];
+}
+
+export interface IUtenlandsopphold {
+    fraDato: string;
+    tilDato: string;
+    årsak: string;
+}
+
+export interface IOppholdstatus {
+    fraDato?: string;
+    tilDato?: string;
+    oppholdstillatelse: Oppholdstatus;
+}
+
+export const enum Oppholdstatus {
+    MIDLERTIDIG = 'MIDLERTIDIG',
+    PERMANENT = 'PERMANENT',
+    UKJENT = 'UKJENT',
+}
```

Comment:
Sjekk ut trikset for å gjøre om enum til "normal tekst" fra Oppgavebenken - eks. `oppgavetema.ts`:

```export const oppgaveTypeTilTekst: Record<Oppgavetype, string> = {
    BEH_SAK: 'Behandle sak',
     ......
```

Reply:
Nice, jag tror det var dette My prøvde å finne når vi satte opp dette men att vi ikke fant det

Reply:
84b753b